susemanager-utils/susemanager-sls/susemanager-sls.spec

```diff
@@ -47,6 +47,14 @@ BuildArch:      noarch
 Static Salt state files for SUSE Manager, where generic operations are
 provided for the integration between infrastructure components.
 
+%package -n susemanager-config-modules
```

Comment:
@juliogonzalez is there any plan to rename `susemanager-sls` to `uyuni-sls` or anything else more "neutral"?

Here we are creating a subpackage with "susemanager" in the name - but we could as well use a different name if you think it'd be more appropriate

Reply:
Not for now. Renaming packages is always a PITA, as you will know, and @mcalmer can tell you :-)

So I'd not do it unless we have a very strong need.

Ad personally I'd call it `susemanager-sls-config-modules`. @mcalmer what's your opinion?

Reply:
My problem with the `-sls` part is that SLS is the (typical) filename extension to Salt states, which is not really the content of this package (which is about modules, written in Python top to bottom).

So really this is a misnomer for the whole `susemanager-sls` package - it used to contain SLS files alone, but has modules since more than a while. And the subpackage under discussion ships modules exclusively and no SLS files (if we exclude examples for documentation purposes).

So I'd suggest going with the currently proposed name unless there are reasons I am not taking into account...

Reply:
For new packages we should really think about using neutral names. So uyuni-something would be preferend from my point of view. Also when it is only a "subpackage" like in this case.

So `uyuni-config-modules` would be ok for me. 

Renaming existing packages is not easy and result always in troubles.